src/richie/apps/core/templates/richie/web_analytics.html

```diff
@@ -1,29 +1,64 @@
-{% block web_analytics %}
-    {% if WEB_ANALYTICS_ID %} 
-        {% if WEB_ANALYTICS_PROVIDER == "google_tag_manager" %}
-            <script async src="https://www.googletagmanager.com/gtag/js?id={{ WEB_ANALYTICS_ID | safe }}"></script>
-            <script>
-                window.dataLayer = window.dataLayer || [];
-                function gtag(){dataLayer.push(arguments);}
-                gtag('js', new Date());
-                {% for dimension_key, dimension_value_list in WEB_ANALYTICS_DIMENSIONS.items %}
-                    gtag('set', {'dimension{{forloop.counter}}': '{{ dimension_value_list|join:" | " }}'});
-                {% endfor %}
-                gtag('config', '{{ WEB_ANALYTICS_ID | safe }}');
-            </script>
-        {% endif %}
-        {% if WEB_ANALYTICS_PROVIDER == "google_analytics" %}
-            <script>
-                (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-                (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-                })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-                ga('create', '{{ WEB_ANALYTICS_ID | safe }}', 'auto');
-                {% for dimension_key, dimension_value_list in WEB_ANALYTICS_DIMENSIONS.items %}
-                    ga('set', {'dimension{{forloop.counter}}': '{{ dimension_value_list|join:" | " }}'});
-                {% endfor %}
-                ga('send', 'pageview');
-            </script>
+
+{% comment %}
+Template parameter:
+ - `location`: where to put the web analytics content. Possible values `head` or `footer`.
+{% endcomment %}
+
+{% for provider, provider_configuration in WEB_ANALYTICS.items %}
+    {% if provider_configuration.tracking_id %}
+        {% if provider_configuration.location == location or location == 'head' and provider_configuration.location is None %}
```

Comment:
For performance concerns, what do you think to put those scripts into the footer by default ?
```suggestion
        {% if provider_configuration.location == location or location == 'footer' and provider_configuration.location is None %}
```

Reply:
Agree.
This is something that depends on the web analytics js code, the site,if you accept to not track some traffic, etc.
But I agree that by default we should put it in the footer. I'm going to also add a new line in the `CHANGELOG.md` file to warn about the change of behavior.